app/services/revocation_registry.py

```diff
@@ -181,19 +197,19 @@ async def publish_pending_revocations(
             f"Failed to publish pending revocations: {e.detail}", e.status_code
         ) from e
 
-    if not result.txn or not result.txn.transaction_id:
+    if not result.txn or not result.txn[0].transaction_id:
         bound_logger.warning(
             "Published pending revocations but received no endorser transaction id. Got result: {}",
             result,
         )
         return
 
-    endorse_transaction_id = result.txn.transaction_id
-    bound_logger.debug(
+    endorse_transaction_id = result.txn[0].transaction_id
```

Comment:
hmm ... looks like we need to now endorse a potential list of transaction ids? Here we're just taking the first one